lbrynet/lbrynet_daemon/auth/server.py

```diff
@@ -255,20 +272,8 @@ def _callback_render(self, result, request, id, version, auth_required=False):
             encoded_message = jsonrpclib.dumps(result_for_return, version=version, default=default_decimal)
             self._set_headers(request, encoded_message, auth_required)
             self._render_message(request, encoded_message)
-        except:
-            fault = jsonrpclib.Fault(self.FAILURE, "can't serialize output")
-            encoded_message = jsonrpclib.dumps(fault, version=version)
-            self._set_headers(request, encoded_message)
-            self._render_message(request, encoded_message)
-
-    def _errback_render(self, failure, id):
-        log.error("Request failed:")
-        log.error(failure)
-        log.error(failure.value)
-        log.error(id)
-        if isinstance(failure.value, jsonrpclib.Fault):
-            return failure.value
-        return server.failure
+        except Exception as err:
```

Comment:
log.exception already includes the traceback so `err.message` isn't necessary.  Its better to instead add some more context, like: `log.exception('failed to render results: %s', result)`